app/src/main/kotlin/com/wire/android/ui/settings/devices/DeviceDetailsScreen.kt

```diff
@@ -187,17 +188,19 @@ fun DeviceDetailsContent(
                     Divider(color = MaterialTheme.wireColorScheme.background)
                 }
             }
-            item {
-                EndToEndIdentityCertificateItem(
-                    isE2eiCertificateActivated = state.isE2eiCertificateActivated,
-                    certificate = state.e2eiCertificate,
-                    isCurrentDevice = state.isCurrentDevice,
-                    isLoadingCertificate = state.isLoadingCertificate,
-                    enrollE2eiCertificate = { enrollE2eiCertificate(context) },
-                    updateE2eiCertificate = {},
-                    showCertificate = onNavigateToE2eiCertificateDetailsScreen
-                )
-                Divider(color = colorsScheme().background)
+            appLogger.i("#### ${state.isE2EIEnabled}")
```

Comment:
Maybe forgotten during development?

```suggestion
```

It doesn't say much in its current form, so it wouldn't be useful just seeing `#### true` or `#### false` in the logs.

Reply:
missed it for debugging purposes :D